### DISCUSSION TITLE
Regarding drc violation clean process

### DISCUSSION THREAD

**Discussion by its-rakib** [The-OpenROAD-Project/OpenROAD-flow-scripts#866] [General]
Is it possible  to solve drc of any design in openroad ?
If possible then what's the process of solving drc  in openroad?

**Comment by maliberty** [comment]
Your question is hard to understand.  Would you expand on what you are asking?

**Reply by its-rakib** [reply]
> Your question is hard to understand. Would you expand on what you are asking?
Is it possible to solve drc of any design in openroad ?

**Reply by maliberty** [reply]
What sort of DRC?  There are 100s-1000s of types depending on the PDK.  Are these drcs generated by OR or by another tool?  Your question is too broad to be meaningfully answered.

**Reply by its-rakib** [reply]
i mean what's the process? how do i solve drc using klayout? do i need to go back to openroad or can i solve it completely in klayout? in that case how to integrate it in flow?

**Reply by maliberty** [reply]
Were drcs reported during detailed routing?  Are they are the base layers or the routing layers?  

You can always fix things manually by editing the gds but it isn't fun.
